akka-http/src/main/java/akka/http/javadsl/server/directives/ContentTypeResolver.java

```diff
@@ -0,0 +1,29 @@
+package akka.http.javadsl.server.directives;
```

Comment:
I added it, please remember for next time - easiest to set up intellij to add license headers to new files